collections/_article/introducing-new-forum.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Introducing the new Godot Forum"
+excerpt: "We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine."
+categories: ["news"]
+author: Winston Yallow
+image: /storage/blog/covers/introducing-new-forum.webp
+date: 2023-12-09 00:00:00
+---
+
+For the last couple of months, our Q&A platform was in readonly mode. Now it has been superceded by our new and more powerful forum! 🎉
+
+🔗 [https://forum.godotengine.org/](https://forum.godotengine.org/)
+
+And don't worry: no content is lost, all the old posts have been migrated to the new platform.
+
+## Why did we replace the old platform?
+
+The old Q&A platform was based on Question2Answer, an open source project for Q&A sites. Unfortunately it is only infrequently updated and had some security issues we couldn't easily solve. This led to problems like bot registrations and spam.
+
+On top of that, we also wanted more forum features that Question2Answer simply couldn't provide. It served it's purpose well for quite some time, but we have simply outgrown it's capabilities. So it's time to give our community a new home!
+
+## Introducing Discourse
+
+The new forum is based on [Discourse](https://www.discourse.org/), an open source forum software that can easily be adjusted via plugins and themes to suit our needs. For logins, the same SSO (single sign on) is used as for other Godot platforms (like the developer chat). This allows users to have one account accross all Godot websites.
+
+## Data Migration
+
+Let's look at some details about our process going from Question2Answer to Discourse!
+
+The old Q&A platform had over 150k questions, answers and comments stored in the database. 20% of those were hidden by our moderators for being spam or breaking the rules, so we ended up importing 120k items from the old database.
+
+What we did not import were the users. The old instance had lot's of old and unused accounts, and it would have been a lot of work to reconcile those users with the users of the SSO server. Instead, all imported posts are made by a system user, and the original username is shown at the top of the post. Since discourse does not support comments on posts like Question2Answer, we embed them directly within the post and add some styling. This way we can represent all data from the old platform in a neat way:
```

Comment:
```suggestion
What we did not import were the users. The old instance had lots of old and unused accounts, and it would have been a lot of work to reconcile those users with the users of the SSO server. Instead, all imported posts are made by a system user, and the original username is shown at the top of the post. Since Discourse does not support comments on posts like Question2Answer, we embed them directly within the post and add some styling. This way we can represent all data from the old platform in a neat way:
```